src/features/clusters/currentClusterSlice.ts

```diff
@@ -32,12 +34,19 @@ export const currentClusterSlice = createSlice({
   reducers: {
     updateCluster: (state, action: PayloadAction<Cluster>) => ({ ...state, data: action.payload }),
     cleanCluster: () => initialState,
+    forceReload: (state, action: PayloadAction<boolean>) => ({
```

Comment:
how about turning this into 2 actions: `startPolling` and `stopPolling`? It would IMO make the code more understandable

Reply:
I can split into two actions but named `forceReload` and `cancelReload`. The original idea behind this PR was to provide an option how to "speed-up" polling from outside (like after an action not returning data update or similar).